ui-modules/utils/quick-launch/quick-launch.js

```diff
@@ -150,8 +150,8 @@ export function quickLaunchDirective() {
 
         function deployApp() {
             $scope.deploying = true;
-
-            Promise.resolve(quickLaunch.buildYaml())
+            let useExtensions = $scope.app.plan.format.includes('camp') ? false : true;
+            Promise.resolve(quickLaunch.buildYaml(useExtensions))
```

Comment:
`includes` returns boolean so much less redundant to simply do:

Promise.resolve(quickLaunch.buildYaml(!$scope.app.plan.format.includes('camp')))

Pretty good otherwise

Reply:
addressed the comment. Thanks